src/compute/src/logging/compute.rs

```diff
@@ -355,6 +373,20 @@ pub(super) fn construct<A: Allocate + 'static>(
     })
 }
 
+/// Format the given value and pack it into a `Datum::String`.
```

Comment:
If we find more uses for that maybe we can move this to `mz_ore` as a newtype that wraps a `String` and has an `as_str(&mut self, value: V): &str` method.